packages/teleport/src/Main/Main.tsx

```diff
@@ -26,12 +26,13 @@ import cfg from 'teleport/config';
 import SideNav from 'teleport/SideNav';
 import TopBar from 'teleport/TopBar';
 import { BannerList } from 'teleport/components/BannerList';
-import getFeatures from 'teleport/features';
```

Comment:
unless i'm missing something, we have two `getFeatures`, this one returns OSS features, the `e-teleport/features` returns OSS features + enterprise features

Reply:
I've changed these to named exports, so we don't confuse the two of them.